### ISSUE TITLE
comment attached to hat block to high imho

### ISSUE THREAD

**Issue by brianharvey** [jmoenig/Snap--Build-Your-Own-Blocks#1651]
When you attach a comment to a hat block, its vertical position lines up with the vertical center of the hat block.  But that's just over the top of the non-swooshy part, so it really doesn't look very firmly attached.  This is a super minor point, but it'd be better centered on the horizontal part.
![hat-comment](https://cloud.githubusercontent.com/assets/3885074/22623325/9a8742e4-eb0b-11e6-91e8-0e5e5c7979c8.png)

**Comment by liam4** [comment]
Agreed. Custom blocks have the same issue, which is particularly annoying..

![Custom-block image](https://cloud.githubusercontent.com/assets/9948030/22623450/f15eee8e-eb31-11e6-87fd-646b1110ff0e.png)


(PS: Are you implementing a specification? That 5.2.1.1e-f sure reminds me of indexes of things like the ECMAScript spec..)

(PSS: Library importer is still wonderful!)

**Comment by brianharvey** [comment]
Solutions to exercises 1e-f in Unit 5, Lab 2, page 1.  :)

(PS:  You mean "PPS."  Post post scriptum.  After after the writing. :P)

**Comment by brianharvey** [comment]
![reporter-comment](https://cloud.githubusercontent.com/assets/3885074/22623915/caa3166a-eb1f-11e6-9bb1-a0172e705ba3.png)
Now that I'm sensitized to it, these ovals don't line up either.  (And the fact that the text is below center in the comment block exacerbates the problem.)

**Comment by cycomachead** [comment]
> Now that I'm sensitized to it, these ovals don't line up either. (And the fact that the text is below center in the comment block exacerbates the problem.)

I think you might have a browser settings problem with that last image, because "age" is also not centered properly and is cut off. I'm not sure what the fix is, but most non-default fonts or font sizes mess with morphic's alignment.

**Comment by brianharvey** [comment]
Maybe the font, but not the oval heights.  Here's my (vanilla) Chrome:
![chrome-ovals](https://cloud.githubusercontent.com/assets/3885074/22624043/d9fc22e6-eb24-11e6-815e-e6669ad61565.png)

**Comment by liam4** [comment]
(Or.. perhaps a light shadow? Since typically you put your notes above or attached to things, not below, and it'd help with the entire general visibility of the comment.)

**Comment by cycomachead** [comment]
> Maybe the font, but not the oval heights. Here's my (vanilla) Chrome:

Ah, you're right. The funny thing about you're original image is that when I zoomed in, the little yellow line looked like it was aligned with the 'e' fairly well. Also, if you drop a comment on a predicate block it's clear that it's attached below the middle, since there's a nice point to it. 

I also 👍 thickening that line to 2px, which is probably enough.
